### DISCUSSION TITLE
Employee Experience Base

### DISCUSSION THREAD

**Discussion by bwatson23** [microsoft/Templates-for-Power-Platform#68] [General]
Hi Team,

I installed the Kudo's app from AppSource and tried to manually install the Expense Reimbursement app taking the manual package from Git.

The Employee Experience version appears to be out of sync between the two 1.0.0.16 was installed but Expense Reimbursement has a dependency on 1.0.0.19 and thus can't install.

Kind regards,
BW

**Comment by ojsauza** [comment]
<img width="478" alt="PowerPlatformError" src="https://github.com/user-attachments/assets/0c2122ac-cc50-4e53-933c-66e46ebd7bf8">

The same issue with install for me. Additionally, the Managed install file for Expense Reimbursement in the [Microsoft solution documentation](https://learn.microsoft.com/en-us/power-platform/enterprise-templates/hr/expense-reimbursement/install-and-set-up#step-4-install-solution-files) is broken. Points to: https://aka.ms/mpa_ExpenseReimbursement_managed.zip
